integration/main_test.go

```diff
@@ -208,6 +208,13 @@ func WithPodLabels(kvs map[string]string) PodSandboxOpts {
 
 // PodSandboxConfig generates a pod sandbox config for test.
 func PodSandboxConfig(name, ns string, opts ...PodSandboxOpts) *runtime.PodSandboxConfig {
+	var cgroupParent string
+	runtimeConfig, _ := runtimeService.RuntimeConfig(&runtime.RuntimeConfigRequest{})
+	if runtimeConfig != nil &&
+		runtimeConfig.GetLinux() != nil &&
+		runtimeConfig.GetLinux().GetCgroupDriver() == runtime.CgroupDriver_SYSTEMD {
+		cgroupParent = "/test.slice"
```

Comment:
Suggest to define a const with`/containerd-test.slice`.

Reply:
And we need to ensure `runc` is also using systemd cgroup as well.

Reply:
We can consider to define new matrix test for systemd cgroup.

Reply:
> Suggest to define a const with`/containerd-test.slice`.

agree.

> And we need to ensure runc is also using systemd cgroup as well.

current only change CRI level, runc should ask users to make sure systemd is used.

> We can consider to define new matrix test for systemd cgroup.

yes, this pr https://github.com/containerd/containerd/pull/9470 add a systemd cgroup matrix, eg: `cgroup_driver: [cgroupfs, systemd]`